test/system/280-update.bats

```diff
@@ -161,4 +161,552 @@ device-write-iops   = /dev/zero:4000 | - | -
     run_podman rm -f -t0 testctr
 }
 
+function _create_container_check_change_of_HealthCheck_configuration_with_podman_update(){
+    local ctrname="$1"
+    local msg="$2"
+    local format="$3"
+    local flag="$4"
+    local value="$5"
+    local expect="$6"
```

Comment:
There's almost no need for this one, either. It is almost always identical to `$value`. Therefore, the logic should be: 
```bash
    local expect="${6:-$5}"
```
so the caller can then use `""` to reduce duplication.